--- ncmpc-0.12~beta1/configure.ac~	2008-11-25 21:36:07.000000000 +0100
+++ ncmpc-0.12~beta1/configure.ac	2008-11-26 12:37:49.000000000 +0100
@@ -231,19 +231,6 @@
 
 dnl CFLAGS
 
-AC_SUBST(AM_CFLAGS)
-
-CHECK_CFLAG([-Wall])
-CHECK_CFLAG([-Wextra])
-CHECK_CFLAG([-Wno-deprecated-declarations])
-CHECK_CFLAG([-Wmissing-prototypes])
-CHECK_CFLAG([-Wdeclaration-after-statement])
-CHECK_CFLAG([-Wshadow])
-CHECK_CFLAG([-Wpointer-arith])
-CHECK_CFLAG([-Wstrict-prototypes])
-CHECK_CFLAG([-Wcast-qual])
-CHECK_CFLAG([-Wwrite-strings])
-
 dnl Optional screen - help screen
 AC_MSG_CHECKING([whether to include the help screen])
 AC_ARG_ENABLE([help-screen],
